frontend/src/pages/Upload.tsx

```diff
@@ -107,9 +132,9 @@ const Upload = () => {
 
   const handleSubmit = async (e: React.FormEvent) => {
     e.preventDefault();
-    
+
     if (!selectedImage) {
-      toast.error('Please select an image');
+      toast.error('Please select a file');
```

Comment:
This message shouldn't be changed. I understand you want to support the upload of PDF. That is ok. But still the PDF must be referring to an image (a photo or a drawing). So, "an image" is correct. This is "image" more in English, rather than the CS definition of that.